### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by youhanaNaseim**
This change is [Reviewable](https://reviewable.io/reviews/youhananaseim/simpleform/6)

### CODE REVIEW

File: simpleFormAdminApp/simpleFormAdminApp/App.xaml.cs [hunk 1]
```diff
@@ -15,3 +15,4 @@ public partial class App : Application
     {
     }
```
Comment (youhanaNaseim) [inline simpleFormAdminApp/simpleFormAdminApp/App.xaml.cs]: @mmitrik fdlksjfsd slfjsdkfj sld fsjfsj slfs`if hello`

@mmitrik

Comment (mmitrik) [inline simpleFormAdminApp/simpleFormAdminApp/App.xaml.cs]: replying

**Comment by mmitrik** [comment]
another reply

**Comment by mmitrik** [comment]
this is _bold_

**Pull request closed without merging**